JSDemos/Demos/DataGrid/ColumnTemplate/description.md

```diff
@@ -1 +1,3 @@
-The **DataGrid** widget supports custom column templates. In&nbsp;this demo, a&nbsp;custom column template is&nbsp;assigned to&nbsp;the **cellTemplate** option of&nbsp;the _Picture_ column.
\ No newline at end of file
+**DataGrid** columns support custom templates to display any content.
+
+In this demo, a custom column template is assigned to the [cellTemplate](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#cellTemplate) ([cellRender](/Documentation/ApiReference/UI_Widgets/dxCalendar/Configuration/#cellRender) in React) option of the `Picture` column to display the employee's image.
```

Comment:
1. This sentence is definitely too long and complicated. It's better to divide it in two.

2. The construction 'a custom column template is assigned to the **cell**Template' seems a bit weird to me. But it might be subjective.

Reply:
I would suggest moving the info about the option out of this sentence because `cellTemplate` is used not only in this demo. Somewhere above should be a sentence like "Use the cellTemplate option to specify a template for column cells."